src/data-input.js

```diff
@@ -0,0 +1,26 @@
+import React, {PropTypes as t} from 'react';
+
+const DataInput = ({autoFocus, name, onChange, onKeyPress, value, label}) => (
+  <span>
+    <label>{label}</label>
+    <input
+      autoFocus={autoFocus}
+      name={name}
+      onChange={onChange}
+      onKeyPress={onKeyPress}
+      type="text"
+      value={value}
+    />
+  </span>
+);
+
+export default DataInput;
```

Comment:
I think it's recommended for the default export to be the last line in the file.